### ISSUE TITLE
Getting Error running plugins store

### ISSUE THREAD

**Issue by sjbbaggie** [MscrmTools/XrmToolBox#599]
![pluginstore](https://cloud.githubusercontent.com/assets/26407319/23898009/4135fcee-08a7-11e7-85ca-b0f23eaca263.PNG)
when ever I run the plugin store I get an error
"The module was expected to contain an assembly manifest (Exception from HRESULT: 0x80131018)"

**Comment by MscrmTools** [comment]
Hi,
Never seen it before... can you confirm you unblocked XrmToolBox.zip file before extracting it ?

**Comment by sjbbaggie** [comment]
this doesn't make any difference unfortunately.

**Comment by MscrmTools** [comment]
what is the version of XrmToolBox?

**Comment by sjbbaggie** [comment]
v1.2017.2.13
I should point out that the very first time I downloaded it , I was able to run plugins store but since then I keep getting the message.  Have tried many times to re-download and run but keep getting the same problem.  I have Windows 10 Pro

**Comment by MscrmTools** [comment]
Can you click on the last button on the plugins store toolbar to clear the cache. Then restart XTB and try again

**Comment by sjbbaggie** [comment]
the green bar at the bottom gets so far and then I get the error

**Comment by sjbbaggie** [comment]
sorry - that hasn't fixed it either

**Comment by MscrmTools** [comment]
ok, next test. 

Can you create a shortcut for XrmToolBox.exe, then edit the shortcut to add /overridepath:<a path to a folder on your desktop> (like /overridepath:c:\users\mylogin\desktop\test)

Then run from the shortcut

Does the error still occur?

**Comment by sjbbaggie** [comment]
I got a different error - but can you clarify the format of the overridepath?  my login is xxxx - so I used /overridepath:c:\users\xxxx\desktop\test - is that right?

**Comment by MscrmTools** [comment]
yes that should do the trick
even simplier, use /overridepath:. (but it will create some folders on the same location than XrmToolBox)

**Comment by sjbbaggie** [comment]
ok - great that worked -however there's one plugin that is showing in red
 Attribute Manager  1.2017.3.16 BETA! Used to rename/delete/or change the type of an attribute...

Many thanks!

**Comment by MscrmTools** [comment]
For information, that just means that something is wrong with your XrmToolBox default storage (in c:\users\xxxx\appdata\roaming\MscrmTools\XrmToolBox).

The plugin in read is badly configured. I will see with the developer

**Comment by DiggaTheWolf** [comment]
I've just hit this same issue after performing an updates and installation of new plugins.

I went through each of the plugins one by one sorted by modified date and manually deleted the dll. Restarted XrmToolBox after each deletion, until I didn't get the exception. In my case this was a problem with the Entity 360 View plugin. I restored all other plugins from the recycle bin and XrmToolBox functions without issue.
